# Window Properties
window_x: 0
window_y: 0
window_width: 700
window_height: 400
